tox.ini

```diff
@@ -0,0 +1,46 @@
+[tox]
```

Comment:
Do we need tox at all? Feels like we can setup airflow_declarative X airflow version matrix on travis without issues.

What I don't like here is that we start to have duplicated definitions of steps to do to make / test our project: travis defines one set of commands, tox - the others, Makefile - one more combo. 

I would prefer that both CI and developer used single set of commands to make things done and that set better define in Makefile. Environment matrix apply via CI.

Reply:
Ok, let's try tox here. Personally, I don't like it, but let's try.